CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to `runme`
+
+**Thank you for your interest in `runme`. Your contributions are highly welcome.**
+
+There are multiple ways of getting involved:
+
+- [Report a bug](#report-a-bug)
+- [Suggest a feature](#suggest-a-feature)
+- [Contribute code](#contribute-code)
+
+## Report a bug
+
+Reporting bugs is one of the best ways to contribute. Before creating a bug report, please check that an [issue](/issues) reporting the same problem does not already exist. If there is such an issue, you may add your information as a comment.
+
+To report a new bug you should open an issue that summarizes the bug and set the label to "bug".
+
+If you want to provide a fix along with your bug report: That is great! In this case please send us a pull request as described in section [Contribute Code](#contribute-code).
+
+## Suggest a feature
+
+To request a new feature you should open an [issue](../../issues/new) and summarize the desired functionality and its use case. Set the issue label to "feature".
+
+## Contribute code
+
+This is an outline of what the workflow for code contributions looks like
+
+- Check the list of open [issues](../../issues). Either assign an existing issue to yourself, or
+   create a new one that you would like work on and discuss your ideas and use cases.
+
+It is always best to discuss your plans beforehand, to ensure that your contribution is in line with our goals.
+
+- Fork the repository on GitHub
+- Create a topic branch from where you want to base your work. This is usually master.
+- Open a new pull request, label it `work in progress` and outline what you will be contributing
+- Make commits of logical units.
+- Make sure you sign-off on your commits `git commit -s -m "adding X to change Y"`
```

Comment:
Maybe we can link to [the GitHub docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) here? This is not so obvious.